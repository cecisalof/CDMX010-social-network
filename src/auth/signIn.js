export const signIn = (container) => {
  const html = `
      <div id="signInContainer">
        <img id="bigSnipple" src="resources/garabato.png" alt="logo" class= "link" data-action = "home">
        <h1 id="signInTitle">Iniciar sesi칩n</h1>
        <form>
          <input type= "text" placeholder="Tu correo electr칩nico"></input>
          <input type= "password" placeholder="Tu contrase침a"></input>
          <button id="signInButton"class="button" data-action="home">Acceder</button>
        </form>
      </div>

      <div class="signInButton">
      <button class="button" data-action="signInWithGoogle">Iniciar sesi칩n con Google</button>
      </div>
    `;
    // eslint-disable-next-line no-param-reassign
  container.innerHTML = html;
};
