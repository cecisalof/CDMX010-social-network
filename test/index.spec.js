// Test de interacción de el flujo de autentificación

// import { login } from '../src/lib/login.js';
import { signIn } from '../src/auth/signIn.js';
// import { signUp } from '../src/lib/signUp.js';
// import { eventsController } from '../src/routes.js';

describe('DOM render', () => {
  // const controller = eventsController();
  it('should show "sign In" view in the DOM', () => {
    const container = document.createElement('div');
    signIn(container);
    //const signInButton = document.querySelector('#signIn');
    //signInButton.click();
    expect(container.innerHTML).toMatchSnapshot();
  });
});
