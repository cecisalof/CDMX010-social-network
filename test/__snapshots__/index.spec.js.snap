// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DOM render should show "sign In" view in the DOM 1`] = `
"
        <div id=\\"signInContainer\\">
          <img id=\\"bigSnipple\\" src=\\"resources/garabato.png\\" alt=\\"logo\\" class=\\"link\\">
          <h1 id=\\"signInTitle\\">Iniciar sesi칩n</h1>
          <form>
            <input id=\\"userEmail\\" type=\\"text\\" placeholder=\\"Tu correo electr칩nico\\">
            <input id=\\"userPassword\\" type=\\"password\\" placeholder=\\"Tu contrase침a\\">
            <button class=\\"button\\" data-action=\\"signInUser\\">Acceder</button>
          </form>
        </div>
        <div class=\\"signInButton\\">
        <button id=\\"googleSignInButton\\" class=\\"button\\" data-action=\\"signInWithGoogle\\">Iniciar sesi칩n con Google</button>
        </div>
      "
`;
